ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -239,6 +239,107 @@ device_classes: !ZenPackSpec
                                 rrdmin: 0
                             missing:
                                 rrdmin: 0
+                graphs:
+                    Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            size:
+                                dpName: volume_size
+                                lineType: DONTDRAW
+                            free:
+                                dpName: volume_free
+                                lineType: AREA
+                                rpn: "size,-,size,/,-100,*"
+                thresholds:
+                    unallocatable:
+                        dsnames: [status_allocatable]
+                        minval: 1
+                        eventClass: /Status
+                    exported:
+                        dsnames: [status_exported]
+                        maxval: 0
+                        severity: 2
+                        eventClass: /Status
+                    missing:
+                        dsnames: [status_missing]
+                        maxval: 0
+                        eventClass: /Status
+            VolumeGroup:
+                description: Volume Group Monitoring
```

Comment:
It'd probably be a good idea to refer to LVM in the descriptions of these monitoring templates. It's a very useful piece of context that is otherwise not particularly obvious to someone not familiar with the implementation.